app/controllers/steps/disbursement_type_controller.rb

```diff
@@ -20,10 +20,7 @@ def decision_tree_class
     end
 
     def disbursement
-      @disbursement ||= begin
-        disbursement_id = params[:disbursement_id]
-        current_application.disbursements.find_by(id: disbursement_id)
-      end
+      @disbursement ||= current_application.disbursements.find_by(id: params[:disbursement_id])
     end
 
     def ensure_disbursement
```

Comment:
this needs to redirect to disbursement_cost_form on save and continue

Reply:
That is handled elsewhere... as the routes for this page are:
```
/applications/{application_id}/steps/discursement_type/{disbursement_id}
```
we need to select the specific disbursement for use in the form... and that is what this bit of code does here...